Config/Inc/config.h

```diff
@@ -33,6 +33,10 @@ typedef enum SafetyStatus_e {SAFE = 0, DANGER = 1, OVERVOLTAGE = 2, UNDERVOLTAGE
 #define NUM_BATTERY_MODULES				31		// Number of battery modules
 #define NUM_TEMPERATURE_SENSORS			62		// Number of temperature sensors
 #define NUM_TEMP_SENSORS_PER_MOD		2		// Number of temperature sensors per battery module
+#define NUM_BATTERY_MODULES_PER_MINION	8	// User defined. The LTC6811 can actually measure 12 modules.
+// Number of battery modules less than maximum value per minion board
+#define NUM_BATTERY_MODULES_MISSING     NUM_BATTERY_MODULES - ((NUM_MINIONS-1) * NUM_BATTERY_MODULES_PER_MINION)
+#define OPEN_WIRE_MASK                  0x7F    // Mask used to ignore disconnected battery modules in Open Wire Test
```

Comment:
Is this used anywhere?

Reply:
I am thinking that we only need line 38 or 39. If we use 38, then we need that code in the init function above. Or we can use line 39 and get rid of that code that creates the mask. I personally prefer having line 39 and getting rid of line 38 because it would be easier to understand later on.

Reply:
With the addition of `open_wire_mask` to Voltage.c, shouldn't we remove `OPEN_WIRE_MASK`?

Reply:
Removed it